libs/sdk-ui-theme-provider/src/cssProperties.ts

```diff
@@ -113,6 +114,14 @@ const generateComplementaryPalette = (palette: IThemePalette): CssProperty[] =>
     );
 };
 
+export const generateShadowColor = (palette: IThemePalette, isDarkTheme: boolean) => {
+    if (!palette?.complementary) {
+        return [];
+    }
+
+    return [getCssProperty("shadow-color", isDarkTheme ? BLACK_COLOR : palette.complementary.shade8)];
```

Comment:
I might be wrong here. But is `palette.complementary` already filled with all the generated colors? If not, this would fail.